### DISCUSSION TITLE
✨ Import an image from host and use it as base image to build from a Dockerfile

### DISCUSSION THREAD

**Discussion by dirodriguezm** [dagger/dagger#12153] [Ideas]
### What are you trying to do?

The goal would be to import an image using `dag.Container().Import(baseImage)` and use that `baseImage` in a `FROM` statement in a Dockerfile.

### Why is this important to you?

Sometimes, instead of rewriting a Dockerfile using Dagger SDK, it's easier to just build from a Dockerfile. But if this Dockerfile uses a base image that's only available locally, then it can't be built, since the local image is not available to the Dagger engine.

### How are you currently working around this?

I think either rewriting the Dockerfile using the Dagger SDK instead or putting the local image on a remote registry should work.

**Comment by RyanGuild-us-ignite** [comment]
This is important to me as an ops person because often the software I need to build comes with a Dockerfile that I do not control. Even if this is a Docker anti-pattern, Dagger should support it because it's a sharp edge for users: 

even if I `export-image`, I can't use it in a build.
